hosted/aks/p1/p1_import_test.go

```diff
@@ -78,6 +74,9 @@ var _ = Describe("P1Import", func() {
 		})
 
 		It("should be able to update cluster monitoring", func() {
+			if helpers.SkipUpgradeTests {
+				Skip("Skipping test for v2.8 ...")
```

Comment:
Perhaps make this message a constant in `structs.go`?
```suggestion
				Skip("Skipping upgrade tests since only one minor k8s version is supported by the current rancher version...")
```

Reply:
Add log as constant.
`Skip` can be called in only Setup or Subject node closure.